tests/unit/descriptor_buffer.cpp

```diff
@@ -744,6 +745,24 @@ TEST_F(NegativeDescriptorBuffer, BindingAndOffsets) {
                 command_buffer.end();
             }
         }
+
+        {
+            const VkDescriptorSetLayoutBinding binding = {0, VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER, 1, VK_SHADER_STAGE_VERTEX_BIT,
+                                                          nullptr};
+            const auto set_layout_no_flag = VkDescriptorSetLayoutObj(m_device, {binding});
+            const auto pipeline_layout_2 = VkPipelineLayoutObj(m_device, {&set_layout_no_flag, &set_layout_no_flag});
```

Comment:
```suggestion
            const VkDescriptorSetLayoutObj set_layout_no_flag(m_device, {binding});
            const VkPipelineLayoutObj pipeline_layout_2(m_device, {&set_layout_no_flag, &set_layout_no_flag});
```
But maybe there is a reason to write it the current way?

Reply:
no specific reason, suggested version is cleaner